contracts/Claims.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity ^0.7.5;
+
+// @notice Used to create Radicle Link identity claims
+contract Claims {
+    /// @notice Emitted on every Radicle Link identity claim
+    /// @param addr The account address which made the claim
+    event Claimed(address indexed addr);
+
+    /// @notice Creates a new claim of a Radicle Link identity.
+    /// Every new claim invalidates previous ones made with the same account.
+    /// The claims have no expiration date and don't need to be renewed.
+    /// If either `version` is unsupported or `payload` is malformed as per `version`,
+    /// the previous claim is revoked, but a new one isn't created.
+    /// Don't send a malformed transactions on purpose, to properly revoke a claim see `version`.
+    /// @param version The version of `payload` encoding, currently supported values:
+    /// - `1` - `payload` is exactly 20 bytes and contains an SHA-1 root hash.
+    /// - `2` - `payload` is exactly 32 bytes and contains an SHA-256 root hash.
+    /// To only revoke a claim without creating a new one, pass root hash `0`,
+    /// which is guaranteed to not match any existing identity.
+    /// @param payload The claim payload
```

Comment:
I find that 'version' is quite confusing. What do you think about renaming it to `enconding` or `format`, which can either be `"sha-1"` or `"sha-256"`. That would reduce indirection both in the naming and in the transcription of this argument.

Reply:
Yeah I was about to say the same, it's more of a "type" or "format" than just version, since it basically describes the content of payload now.

Reply:
I wouldn't use a `string` type however, I think `uint` is fine..

Reply:
I like `format`, fixed.

I too feel like `uint` is better here. It's a little bit cheaper and closer to the EVM metal and a whole class of text formatting and encoding bugs just goes away because integers are unambiguous.